fhirr4/ballerina/src/main/resources/fhirservice/Ballerina.toml

```diff
@@ -20,6 +20,11 @@ org = "ballerinax"
 name = "health.fhir.r4.parser"
 version = "${parser.version}"
 
+[[dependency]]
+org = "ballerinax"
+name = "health.fhir.r4.international401"
+version = "${international401.version}"
```

Comment:
Will this be a issue when releasing fhir service? Instead can't we create a simple test resource record and use it with a test config for the tests?

Reply:
It wouldn't be an issue I suppose. We can just use whatever the released version of international401. There won't be any cyclic dependencies or so.